### PR TITLE
{data,tools}[foss/2020b,gompi/2020b] CDO v1.9.10, YAXT v0.9.0

### PR DESCRIPTION
**Pull request by arielzn**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/c/CDO/CDO-1.9.10-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'CDO'
+version = '1.9.10'
+
+homepage = 'https://code.zmaw.de/projects/cdo'
+description = """CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```
Comment (boegel) [inline easybuild/easyconfigs/c/CDO/CDO-1.9.10-foss-2020b.eb:9]: @arielzn Toolchain can be "lowered" to `gompi`?

**Review by boegel — DISMISSED** [review]

File: easybuild/easyconfigs/c/CDO/CDO-1.9.10-foss-2020b.eb (hunk 1 above)
Comment (arielzn) [reply easybuild/easyconfigs/c/CDO/CDO-1.9.10-foss-2020b.eb]: it seems, indeed

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/fcb1e90f1bfbb399cc2c4c89826ba66f for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/5c7a0fb0baed4dc06e0a5bedb87dc4d1 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @arielzn!

**Pull request merged**